concepts/submission_date.md

```diff
@@ -0,0 +1,77 @@
+# Submission Date
+
+This document will help you understand `submission_date`, why are standardizing on it, and how it
+differs from `activity_date`.
+
+## What are Submission Date and Activity Date
+
+`submission_date` is the server time at which a ping is received from the client. We use it to
+partition many of our data sets.
+
+`activity_date` is the client time at which a subsession started. It is acquired by removing time
+of day from `subsession_start_date`. It is sometimes normalized by throwing away values too far
+from `submission_date`, values not in the years 2000-2099, and values that are not dates.
+
+## Submission Delay
```

Comment:
This is already covered in Analysis Gotchas, so if we combine these docs we can cut this section (or update the other blurb)

Reply:
I moved everything into Analysis Gotchas and updated the Delay blurb within